.github/workflows/clear-environment.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           HOST: ${{ vars.DOMAIN }}
           ENV: ${{ vars.ENVIRONMENT_TYPE }}
           SSH_USER: ${{ secrets.SSH_USER }}
-          SSH_HOST: ${{ secrets.SSH_HOST }}
+          SSH_HOST: ${{ vars.SSH_HOST || secrets.SSH_HOST }}
```

Comment:
`SSH_HOST` variable is moved from a secret to a variable. This is for backwards-compatibility